### ISSUE TITLE
podman Failed to Install Ubuntu 20.04 

### ISSUE THREAD

**Issue by aplatypus** [containers/podman#7268]
/kind bug

**Description**

**Steps to reproduce the issue:**

1.  Follow the instruction from the installation page on podman.io
     * https://podman.io/getting-started/installation

2.  use `apt install` command:

```
 $ sudo apt -y install podman
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package podman
```

3.  Double checking ...

```
 $ apt search podman
Sorting... Done
Full Text Search... Done
conmon/focal 2.0.9-1 amd64
  OCI container runtime monitor

resource-agents/focal 1:4.5.0-2ubuntu2 amd64
  Cluster Resource Agents
```
   *  Something to note, the label: "conmon/" above.  
   * Is that a typo in the configuration?  That line is pasted directly from the termina.

**Describe the results you received:**

* Error: "`E: Unable to locate package podman`"
* Can't use or run podman

As an aside, these machines are unable to use Snaps because Snap prevents programs running when there are symbolic links involved.  So we cannot use that either.

**Describe the results you expected:**

* Expected podman to install so that I can get going on my project.
* I doubt that Docker will be enough 

**Additional information you deem important (e.g. issue happens only occasionally):**

This was reported earlier in  

*  [Failed to install podman with download.opensuse.org repo on Travis Ubuntu bionic non x86_64](https://github.com/containers/podman/issues/5447) (5447)

**Output of `podman version`:**

```
(paste your output here)
```

**Output of `podman info --debug`:**

*  N/A can not install

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

*  N/A can not install
*  Podman snap unusable due to symlink bugs
    * https://bugs.launchpad.net/snapd/+bug/1886113
    *  https://forum.snapcraft.io/t/how-can-i-use-snap-when-i-dont-use-home-user/3352/11

**Additional environment details (AWS, VirtualBox, physical, etc.):**

* Physical AMD x86_64
* Ubuntu 20.04 (all packages up to date).

**Comment by aplatypus** [comment]
/update

I believe the problem lies behind the keyboard.  However the documentation for Ubuntu on the install page seems to omit a key step.

* https://podman.io/getting-started/installation#Ubuntu

Therein it has:
```
  :
curl -L https: ...
sudo apt-get update -qq
sudo apt-get -qq -y install podman
```
Which wasn't working for me until I did an `apt upgrade` then I was able to install `podman`

```
sudo apt update
sudo apt upgrade
sudo apt install podman
```

Personally I really don't want "quiet" when I'm loading new untried packages.


Yah!! The correct outcome appears to have occured ...


```
 $ podman info --debug
host:
  arch: amd64
  buildahVersion: 1.15.0
  cgroupVersion: v1
  conmon:
    package: 'conmon: /usr/libexec/podman/conmon'
    path: /usr/libexec/podman/conmon
    version: 'conmon version 2.0.18, commit: '
  cpus: 16
  distribution:
    distribution: ubuntu
    version: "20.04"
  eventLogger: file
  hostname: myhost
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.4.0-42-generic
  linkmode: dynamic
  memFree: 8074997760
  memTotal: 33687072768
  ociRuntime:
    name: runc
    package: 'runc: /usr/sbin/runc'
    path: /usr/sbin/runc
    version: 'runc version spec: 1.0.1-dev'
  os: linux
  remoteSocket:
    path: /run/user/1000/podman/podman.sock
  rootless: true
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 1.1.4
      commit: unknown
      libslirp: 4.2.0
      SLIRP_CONFIG_VERSION_MAX: 2
  swapFree: 2081693696
  swapTotal: 2147479552
  uptime: 292h 49m 12.3s (Approximately 12.17 days)
registries:
  search:
  - docker.io
  - quay.io
store:
  configFile: /home/user/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: vfs
  graphOptions: {}
  graphRoot: /home/user/.local/share/containers/storage
  graphStatus: {}
  imageStore:
    number: 0
  runRoot: /run/user/1000/containers
  volumePath: /home/user/.local/share/containers/storage/volumes
version:
  APIVersion: 1
  Built: 0
  BuiltTime: Thu Jan  1 10:00:00 1970
  GitCommit: ""
  GoVersion: go1.14.2
  OsArch: linux/amd64
  Version: 2.0.4
```

In summary there's one more step needed, it seems.

**Comment by mheon** [comment]
Would you mind making a PR to update the install page with this? It lives at https://github.com/containers/podman.io/blob/master/getting-started/installation.md
